### DISCUSSION TITLE
How to get the relationship between topics and documents?

### DISCUSSION THREAD

**Discussion by TheMyNameABC** [MaartenGr/BERTopic#1351] [Q&A]
I want to know all the documents that under a topic.What I should do? I can use which method to do that?

**Comment by MaartenGr** [comment]
You can use `topic_model.get_document_info(docs)` for that. You can find more about BERTopic in the documentation: https://maartengr.github.io/BERTopic/getting_started/quickstart/quickstart.html

**Comment by TheMyNameABC** [comment]
OK~ Thanks. I have solved that problem.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "MaartenGr/BERTopic"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年6月19日(星期一) 中午1:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [MaartenGr/BERTopic] How to get the relationship between topics and documents? (Discussion #1351)





 
You can use topic_model.get_document_info(docs) for that. You can find more about BERTopic in the documentation: https://maartengr.github.io/BERTopic/getting_started/quickstart/quickstart.html

**Reply by MolkaSlama** [reply]
Hello , could you tell me how you solved this issue? Thank you .

**Reply by TheMyNameABC** [reply]
Sorry, I just saw your problem.You can use topic_model.get_document_info(docs) for that. You can find more about BERTopic in the documentation: https://maartengr.github.io/BERTopic/getting_started/quickstart/quickstart.html
